akka-persistence-typed/src/main/scala/akka/persistence/typed/javadsl/CommandHandler.scala

```diff
@@ -79,6 +78,16 @@ final class CommandHandlerBuilder[Command, Event, S <: State, State] @InternalAp
     this
   }
 
+  def matchCommand[C <: Command](commandClass: Class[C], handler: Function[C, Effect[Event, State]]): CommandHandlerBuilder[Command, Event, S, State] = {
+    addCase(cmd ⇒ commandClass.isAssignableFrom(cmd.getClass), (_, cmd) ⇒ handler.asInstanceOf[Function[Command, Effect[Event, State]]].apply(cmd))
+    this
+  }
+
+  def matchCommand[C <: Command](commandClass: Class[C], handler: Supplier[Effect[Event, State]]): CommandHandlerBuilder[Command, Event, S, State] = {
+    addCase(cmd ⇒ commandClass.isAssignableFrom(cmd.getClass), (_, _) ⇒ handler.get())
+    this
+  }
+
```

Comment:
I added some more match methods to cover the cases where we only need a Command to be passed and when we don't need any of them. 

This is to support a one-liner style. Check `AccountExampleOneLiners` below.

Reply:
good, I think I added similar elsewhere